### PR TITLE
Issue 4962 - Fix various UI bugs part 1

### PR DESCRIPTION
**Pull request by mreynolds389**
Fix Description:

Bug 2016022 - Cockpit UI: UI is incorrectly saying "Create the Sub Suffix entry"
Bug 2015951 - Cockpit UI: Database tab ---> Export Database/replicaton
data
Bug 2015221 - Cockpit UI: UX Bugs Server Settings ->Tuning and Limits
Bug 2015139 - Configuration for Import Cache Settings is not saved
Bug 2015127 - No message when configuring Global Database Configuration
Bug 2014924 - Cockpit UI: UX Bugs and other cockpit GUI related defects

relates: https://github.com/389ds/389-ds-base/issues/4962

Reviewed by: ?

### CODE REVIEW

File: src/cockpit/389-console/src/lib/database/databaseConfig.jsx [hunk 1]
```diff
@@ -69,7 +70,9 @@ export class GlobalDatabaseConfig extends React.Component {
             _import_cache_auto: this.props.data.import_cache_auto,
         };
 
+        this.validateSaveBtn = this.validateSaveBtn.bind(this);
```
Comment (droideck) [inline src/cockpit/389-console/src/lib/database/databaseConfig.jsx:73]: When I go to the Database tab -> Click Global Database Configuration - any update doesn't enable 'Save Config' button... It's always grey.

File: src/cockpit/389-console/src/lib/server/settings.jsx [hunk 2]
```diff
@@ -1117,7 +1117,7 @@ export class ServerSettings extends React.Component {
                                         </GridItem>
                                     </Grid>
                                     <Grid
```
Comment (droideck) [inline src/cockpit/389-console/src/lib/server/settings.jsx:1161]: Maybe I missed it, but I don't see if it was addressed here:

"[ 3 ] When updating server settings backup directory error is not displayed to the user" from https://bugzilla.redhat.com/show_bug.cgi?id=2014924

**Review by droideck — CHANGES_REQUESTED** [review]
The rest looks good!

**Review by jchapma — COMMENTED** [review]
2015139
 In Database -> Global Database Configuration. The "Save Config" button doesnt enable when I change any of the settings. 


2014924 - 3
When a non existing back dir is specified by the user an error code is displayed, is it possible to explain the error code.

Screenshot from 2021-10-27 12-29-44


2014924 - 4
Disk Monitoring Grace period grace period error message says operations error. Could we tell the user that the grace period should be a certain value, like the debug console.

Screenshot from 2021-10-27 12-35-32

**Review by droideck — APPROVED** [review]
LGTM!

**Pull request merged**